src/main/java/com/exactpro/th2/codec/csv/CsvCodec.java

```diff
@@ -50,6 +51,9 @@
 public class CsvCodec implements MessageListener<RawMessageBatch> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CsvCodec.class);
+    private static final String HEADER_MSG_TYPE = "CsvHeader";
+    private static final String CSV_MESSAGE_TYPE = "Csv_Message";
```

Comment:
Why there's an underscore in one message type, but not in the other?

Reply:
The `Csv_Message` can be used in other boxes (recon for example). So, I cannot change the message type for it. But I think that the underscore was redundant. That is why I've got rid of it in the message type for the header.

Do you think we should use underscore for consistency?